src/ServerMain.cc

```diff
@@ -196,6 +206,21 @@ main(int argc, char *argv[])
                &config.backup.maxRecoveryReplicas)->default_value(20),
              "Maximum number of replicas any given master recovery will buffer "
              "in memory.")
+            ("minDiskUtilization",
+             ProgramOptions::value<uint32_t>(
+               &config.backup.minDiskUtilization)->default_value(95),
```

Comment:
I'm a little surprised these parameters kept the old high default values.  I though part of the issue was that it didn't start cleaning until rather late in the game.

Reply:
I'm only keeping the older defaults until we figure out what the "new defaults" should be. Then I can change this PR to reflect that.